src/main/java/de/berlin/lostberlin/controller/BusinessController.java

```diff
@@ -13,14 +14,15 @@
 @RequestMapping("/businesses")
 public class BusinessController {
 
+    @Autowired
     private BusinessService businessService;
 
     public BusinessController(BusinessService businessService) {
         this.businessService = businessService;
     }
 
     @GetMapping
-    public ResponseEntity getBusinesses(@RequestParam (required=true, value = "location") String serviceLocation){
+    public ResponseEntity getBusinesses(@RequestParam(required = true, value = "location") String serviceLocation) {
```

Comment:
ResponseEntity  should be parametrised: ResponseEntity<Business>

Reply:
Yeah, I noticed that I forgot to do that right after I created PR and updated it right away (but you were still faster:)